src/foam/u2/md/md.html

```diff
@@ -12,7 +12,7 @@
     <meta name="msapplication-config" content="/favicon/browserconfig.xml">
     <meta name="theme-color" content="#ffffff">
 
-    <script language="javascript" src="../../../foam.js"></script>
+    <script language="javascript" src="../../../foam.js" project="../../../../pom"></script>
```

Comment:
Is the project needed here (and elsewhere)?

Reply:
yes, some demos need nanos and other codes on top of the foam core so project pom is pointing to the pom.js file at the foam3 directory instead of the default pom.js which only contains foam core stuff.